FreeAPS/Sources/Modules/Settings/View/SettingsRootView.swift

```diff
@@ -61,6 +61,9 @@ extension Settings {
                     Text("Dynamic ISF").navigationLink(to: .dynamicISF, from: self)
                 } header: { Text("Extra Features") }
 
+                Section {
+                    Text("Contact trick").navigationLink(to: .contactTrick, from: self)
```

Comment:
right now the Contact Trick section is located in the main settings view, right above the `Debug options` - not sure what's the right place for it